lib/inferno/apps/cli/templates/lib/%library_name%/version.rb.tt

```diff
@@ -1,3 +1,4 @@
 module <%= module_name %>
   VERSION = '0.0.0'.freeze
+  LAST_UPDATED = '1970-01-01'.freeze # TODO: update next release
```

Comment:
I could go either way with what you have or something like 
```suggestion
  LAST_UPDATED = '<%= Date.today.strftime("%Y-%m-%d") %>.freeze # TODO: update next release
```

NIce thing about what you have is its clear its wrong and needs to be updated.  Downside is that we don't really care that much about the last time this thing was updated, and it would be weird to run a test kit that says it was made in 1970 because that's clearly wrong.